### ISSUE TITLE
Master head `rake ..` tasks infinitely recurse on `require 'fhir_client'`. (synthea.rb:12)

### ISSUE THREAD

**Issue by preston** [synthetichealth/synthea#200]
When running `rake <anything>` off master as noted in the instructions, a gnarly recursive stack overflow is emitted. This happens on `rake`, `rake -T`, `bundle exec rake synthea:generate` etc when synthea.rb `requires 'fhir_client'` on line 12.

(additional recursive calls omitted)

> /Users/preston/.rvm/gems/ruby-2.4.0/gems/activesupport-4.1.16/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/activesupport-4.1.16/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/activesupport-4.1.16/lib/active_support/core_ext/numeric/conversions.rb:131:in `block (2 levels) in <class:Numeric>'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/addressable-2.5.1/lib/addressable/version.rb:27:in `join'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/addressable-2.5.1/lib/addressable/version.rb:27:in `<module:VERSION>'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/addressable-2.5.1/lib/addressable/version.rb:22:in `<module:Addressable>'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/addressable-2.5.1/lib/addressable/version.rb:21:in `<top (required)>'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/addressable-2.5.1/lib/addressable/uri.rb:19:in `require'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/addressable-2.5.1/lib/addressable/uri.rb:19:in `<top (required)>'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/fhir_client-3.0.1/lib/fhir_client/client.rb:3:in `require'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/fhir_client-3.0.1/lib/fhir_client/client.rb:3:in `<top (required)>'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/fhir_client-3.0.1/lib/fhir_client.rb:12:in `require_relative'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/fhir_client-3.0.1/lib/fhir_client.rb:12:in `<top (required)>'
/Users/preston/Developer/git/synthea/lib/synthea.rb:12:in `require'
/Users/preston/Developer/git/synthea/lib/synthea.rb:12:in `<top (required)>'
/Users/preston/Developer/git/synthea/Rakefile:5:in `require_relative'
/Users/preston/Developer/git/synthea/Rakefile:5:in `<top (required)>'
/Users/preston/.rvm/gems/ruby-2.4.0/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
/Users/preston/.rvm/gems/ruby-2.4.0/bin/ruby_executable_hooks:15:in `eval'
/Users/preston/.rvm/gems/ruby-2.4.0/bin/ruby_executable_hooks:15:in `<main>'
(See full trace by running task with --trace)
laptop:synthea preston$

**Comment by preston** [comment]
This is happening on ruby v2.4.1 and v2.4.0 on OS X. Interestingly, clone'ing fhir_client and futzing around doesn't seem to have this issue with the underlying calls.

**Comment by jawalonoski** [comment]
I found an issue with `fhir_client` related to `rest-client`. Fixing that with https://github.com/fhir-crucible/fhir_client/pull/68 fixes this issue, but the `fhir_client` gem will need a new release.

There is still an infinitely recursive require, but it is now related to `health-data-standards` gem which is outside of my control. I submitted an issue to that team: https://github.com/projectcypress/health-data-standards/issues/498. It has to do with some of their outdated dependencies.
